reducers/views/search.ts

```diff
@@ -23,6 +23,19 @@ function modalSearch(state = '', action: GenericAction) {
     }
 }
 
+function channelMembersRhsSearch(state = '', action: GenericAction) {
+    switch (action.type) {
+    case SearchTypes.SET_CHANNEL_MEMBERS_RHS_SEARCH: {
+        return action.data.trim();
```

Comment:
I think we'll want to modify this somehow to allow searches for `<firstname> <lastname>`.

https://user-images.githubusercontent.com/13738432/161801424-b390a37a-96a2-44bc-abee-aaaa4a5c623b.mp4